src/preset.ts

```diff
@@ -1020,119 +1148,54 @@ async function tryApplyVsDevEnv(preset: ConfigurePreset) {
                 }
 
                 if (compilerName && whereExecutable) {
+                    // We need to construct and temporarily expand the environment in order to accurately determine if this preset has the compiler / ninja on PATH.
+                    // This puts the preset.environment on top of process.env, then expands with process.env as the penv and preset.environment as the envOverride
```

Comment:
should this be passing in nvironmentUtils.mergePreserveNull([process.env, preset.environment]] as the envorride instead or just preset.environement ? since the fallback behavior in var not in env field is to check parent env

Reply:
Good callout, fixing.

Reply:
Fixed.